packages/bot-web-ui/src/components/dashboard/dashboard-components/index.scss

```diff
@@ -0,0 +1,173 @@
+.bot-dashboard {
+    .toolbar__section {
+        justify-content: end;
+    }
+    .run-panel__container {
+        height: calc(100vh - 5rem) !important;
+        position: unset !important;
+        transform: translateX(calc(350px));
+        transition: all 0.4s;
+        margin-top: 1rem;
+    }
+    .dc-drawer--open {
+        transform: translateX(calc(0px)) !important;
+    }
+}
+.dashboard {
+    &__main {
+        display: flex;
+        overflow: hidden;
+    }
+    &__container {
+        border: none;
+        max-width: 100%;
+        background: var(--general-main-1);
+        width: 100%;
+
+        .dc-tabs {
+            &__active-line {
+                background-color: var(--general-main-1);
+                transition: none;
+            }
+
+            &__active {
+                background: var(--general-main-1);
+            }
+
+            &__list {
+                background: var(--general-section-1);
+                justify-content: start;
+            }
+
+            &__item {
+                border-top-left-radius: 1rem;
+                border-top-right-radius: 1rem;
+                height: 4.8rem;
+                width: 14.4rem;
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                svg {
+                    width: 1.6rem;
+                    height: 1.6rem;
+                    padding-right: 0;
+                    margin-right: 0.8rem;
+                }
+            }
+            &__dashboard-cards-wrapper {
+                display: flex;
+                margin-top: 4rem;
+            }
+
+            &__dashboard-card {
+                margin-right: 4.2rem;
+                display: flex;
+                flex-direction: column;
+
+                svg {
+                    margin-bottom: 0.8rem;
+
+                    &:hover {
+                        transition: all 100ms ease-in;
+                        box-shadow: rgba(0, 0, 0, 0.35) 0px 0.5rem 1.5rem;
+                        border-radius: 0.5rem;
+                        font-weight: 700;
+                    }
+                }
+
+                span {
+                    text-align: center;
+                }
+
+                &:hover {
+                    cursor: pointer;
+                }
+                &:last-child {
+                    margin-right: 0;
+                }
+                &__image {
+                    background: var(--general-section-1);
+                }
+                &__label {
+                    font-size: 1.2rem;
+                    font-weight: 400;
+                    line-height: 2rem;
```

Comment:
Can we use the Text component for the label?

Reply:
done ✅